modules/ROOT/pages/_partials/sync/creating-a-voyager-server-container-image-for-use-with-openshift.adoc

```diff
@@ -0,0 +1,59 @@
+[id="creating-a-voyager-server-container-image-for-use-with-openshift-{context}"]
+= Creating a voyager server container image for use with OpenShift
+
+This section describes how to create a container image using a local voyager server repo.
+
+Although it is not covered in this section, you can add the Apollo Voyager Client.
```

Comment:
why would you do this?
and how?

suggestion: just remove

Reply:
Good spot Paul - it should have stated "Apollo Voyager Server.".

Highlights the value of peer reviews!